thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/DetectionOnboardJob.java

```diff
@@ -0,0 +1,39 @@
+package com.linkedin.thirdeye.anomaly.onboard;
+
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.configuration.Configuration;
+
+public interface DetectionOnboardJob {
```

Comment:
Should the interface restrict to detection onboard job? Do you have plans to extend this design to all jobs in ThirdEye?

Reply:
We could explore the possibility.

In order to extend this temporary executor to TE's generic executor, we need to make sure we can persistent execution status of job and tasks, which shouldn't be too difficult.